text/0011-namespaces.md

```diff
@@ -0,0 +1,30 @@
+# Namespaces 
+- Start Date: 2019/06/17
+- RFC PR: https://github.com/openmrs/openmrs-rfc-frontend/pull/11
+
+## Base Namespace 
+The base namespace for maps and modules will be `@openmrs`
+
+## Core Features 
+The namespace for core features will be `@openmrs/core` which will include core atoms 
+
+## Module based namespaces 
+The namespace for module based features, which will leverage existing modules will be `@openmrs/module/modulename` for example 
+- `@openmr/module/appointmentscheduling` for appointment scheduling 
+
+## Implementing Partner and Distributions 
+The namespace for distributions and partners will be as follows
+- `@openmrs/distro/reference-application/` for the Reference Application 
```

Comment:
`@openmrs/distro/refapp/`?

Reply:
refapp is just a common "nick name", but reference-application is the name of the module and the distribution too. More text to type but more consistent usage across the board 

I am happy to remove the "-", just that I think it aids readability

Reply:
Hmm, this is making me think (in a good way) about what this new reference application would be. The way that I have  been thinking about the new microfrontends architecture is more that the default implementation of each of the core modules would together comprise the equivalent of a ref app. And that the ref app itself wouldn't be a module, but just the set of all core modules.

That line of thinking is shown in [this line of code in openmrs-esm-root-config](https://github.com/openmrs/openmrs-esm-root-config/blob/master/src/root-config-lib.js#L10), where a function is exported that registers *all* the core applications. Also see [this line](https://github.com/openmrs/openmrs-esm-root-config/blob/master/src/root-config-lib.js#L20) and [this file](https://github.com/openmrs/openmrs-esm-root-config/blob/master/src/root-config-dist.js), which try to make it easy for devs who want the default modules and also for those who do not.

What are your thoughts on this? Is there a specific need for a module called ref app, instead of it just being the collective set of core modules?